bundle.Dockerfile

```diff
@@ -14,6 +14,9 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 # Labels for testing.
 LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
+LABEL com.redhat.openshift.versions=v4.10
```

Comment:
v4.10 for the OCP version. will this stay the same until 4.12 is considered?

Reply:
Yes. Since OCP 4.12 is not yet GA and PRD is still not finalized, support matrix will remain as OCP 4.10, OCP 4.11. For this LABEL, we specify the lowest supported version